lib/flipper/ui/configuration.rb

```diff
@@ -9,12 +9,17 @@ class Configuration
                   :percentage_of_actors,
                   :percentage_of_time
 
+      attr_accessor :environment,
+                    :environment_banner_color
+
       def initialize
         @actors = Option.new("Actors", "Enable actors using the form above.")
         @groups = Option.new("Groups", "Enable groups using the form above.")
         @percentage_of_actors = Option.new("Percentage of Actors", "Percentage of actors functions independently of percentage of time. If you enable 50% of Actors and 25% of Time then the feature will always be enabled for 50% of users and occasionally enabled 25% of the time for everyone.") # rubocop:disable Metrics/LineLength
         @percentage_of_time = Option.new("Percentage of Time", "Percentage of actors functions independently of percentage of time. If you enable 50% of Actors and 25% of Time then the feature will always be enabled for 50% of users and occasionally enabled 25% of the time for everyone.") # rubocop:disable Metrics/LineLength
         @delete = Option.new("Danger Zone", "Deleting a feature removes it from the list of features and disables it for everyone.") # rubocop:disable Metrics/LineLength
+        @environment = nil
+        @environment_banner_color = :danger
```

Comment:
Not a blocker for this PR by any means, but what do you think about defining the writer method here and enforcing valid values? Since bootstrap has a defined list of alert- classes, I'm wondering if we should make an array of the valid ones and verify that the one a user is trying to use is valid to avoid confusion if they pick a non-existent one and see no color.

Reply:
Good idea. I'll see what I can come up with.